### PR TITLE
Readme: Mention to uncheck "copy main branch only" on fork

### PR DESCRIPTION
**Pull request by Miepee**
it is really unintuitive that we do not mention this anywhere, and even when it fails we only have a vague 
```
if len(self.randovania_version) != 4:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: object of type 'NoneType' has no len()
```
message.

Theoretically, we could add a mention on how to add upstream and use its tags should one have forgotten to uncheck it, but if one notices it that early they can just delete the fork and refork.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Integration written by [Thanatos](https://github.com/ThanatosGit).
 ## Setup
 
 Getting started:
-   1. Clone this repository. (downloading the zip is *not* supported and will not work)
```
Comment (henriquegemignani) [inline README.md:194]: Don't remove the zip mention. People press the download zip button way too often.

**Review by henriquegemignani — CHANGES_REQUESTED** [review]
I'd rather fix any errors that come from not having tags, but fixing docs helps in the process.

File: README.md (hunk 1 above)
Comment (Miepee) [reply README.md:194]: Huh. Whenever people came into the rdv server for help, I never saw them had issues due to downloading the zip.

Comment (henriquegemignani) [reply README.md:194]: Hopefully because both the script and the read me very explicitly call that out

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**